src/rapidjson/reader.h

```diff
@@ -1210,7 +1210,7 @@ class GenericReader {
         is.dst_ = q;
     }
 
-    // When read/write pointers are the same for insitu stream, just skip unescaped characters
+    // When read/write pointers are the same for in situ stream, just skip unescaped characters
```

Comment:
Thi is the proper spelling for the phrase.

https://en.wikipedia.org/wiki/In_situ

Note that I'm not changingt the class as that seemed a bit too invasive...